src/ngraph/op/strided_slice.cpp

```diff
@@ -56,9 +91,7 @@ op::v1::StridedSlice::StridedSlice(const Output<Node>& data,
     : StridedSlice(data,
                    begin,
                    end,
-                   op::Constant::create(element::i64,
```

Comment:
It was bug found during working on ONNX Slice-10. However, in current implementation of Slice-10 this version of constructor is not used so StridedSlice changes can be reverted in this PR